dsgrid/dataformat/sectordataset.py

```diff
@@ -3,7 +3,6 @@
 from itertools import repeat
 import h5py
 import logging
-from h5pyd._hl.files import is_hdf5
```

Comment:
I think this might be used by one of the notebooks, in which case we could still remove this here but might need to make a code change there?

Reply:
I don't see it in the notebooks, and as I recall h5pyd never worked anyway.

Reply:
setup.py includes
```
oedi_requires = [
    'h5pyd',
    's3fs'
]
```
And it is conditionally imported in H5Reader. So, it should be good now.